app/delivery/meteor_routes.js

```diff
@@ -6,7 +6,12 @@ const meteorRouter = express.Router();
 
 meteorRouter.get('/meteors', async (req, res, next) => {
     try {
-        const meteors = await getMeteors();
+        const {date, count, wereDangerousMeteors} = req.query;
+
+        const isDangerous = wereDangerousMeteors === 'true';
+        const hasCount = count === 'true';
+
+        const meteors = await getMeteors(date, hasCount, isDangerous);
```

Comment:
const meteors = await getMeteors(date, Boolean(count), Boolean(wereDangerousMeteors));

Reply:
But if count=no, then Boolean(count) returns true. Is this the expected behavior?